### PR TITLE
Sample gov script for funding vesters + adjust trading rewards

### PR DESCRIPTION
**Pull request by teddyding**
### Changelist
Add sample script

### Test Plan
Example output:
```
% python3 scripts/governance/bridge_vesters_set_rewards.py
Estimated block height for rewards multiplier update 1 2023-12-19T15:00:00+00:00 = 3609052
Estimated block height for rewards multiplier update 2 2024-01-16T15:00:00+00:00 = 5571096
Output written to proposal_bridge_vesters_set_rewards.json
```

Output file:
```
                "denom": "adv4tnt",
                "denomExponent": -18,
                "marketId": 1000001,
                "fee_multiplier_ppm": 330000
            }
        },
        {
            "@type": "/dydxprotocol.delaymsg.MsgDelayMessage",
            "authority": "dydx10d07y265gmmuvt4z0w9aw880jnsr700jnmapky",
            "msg": {
                "@type": "/dydxprotocol.rewards.MsgUpdateParams",
                "authority": "dydx1mkkvp26dngu6n8rmalaxyp3gwkjuzztq5zx6tr",
                "params": {
                    "treasuryAccount": "rewards_treasury",
                    "denom": "adv4tnt",
                    "denomExponent": -18,
                    "marketId": 1000001,
                    "fee_multiplier_ppm": 660000
                }
            },
            "delay_blocks": 3558938
        },
        {
            "@type": "/dydxprotocol.delaymsg.MsgDelayMessage",
            "authority": "dydx10d07y265gmmuvt4z0w9aw880jnsr700jnmapky",
            "msg": {
                "@type": "/dydxprotocol.rewards.MsgUpdateParams",
                "authority": "dydx1mkkvp26dngu6n8rmalaxyp3gwkjuzztq5zx6tr",
                "params": {
                    "treasuryAccount": "rewards_treasury",
                    "denom": "adv4tnt",
                    "denomExponent": -18,
                    "marketId": 1000001,
                    "fee_multiplier_ppm": 900000
                }
            },
            "delay_blocks": 5466825
        }
    ]
}
```

Tested on a local network 

### Author/Reviewer Checklist
- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
- [ ] If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
- [ ] If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
- [ ] Manually add any of the following labels: `refactor`, `chore`, `bug`.

### CODE REVIEW

File: protocol/scripts/governance/bridge_vesters_set_rewards.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                "denomExponent": -18,
+                "marketId": 1000001,
+                "fee_multiplier_ppm": REWARDS_MULTIPLIER
+            }
+        },
+    ]
+}
+
+# Add delayed messages
+for delay, fee_multiplier in [
```
Comment (ttl33) [inline protocol/scripts/governance/bridge_vesters_set_rewards.py:136]:
```suggestion
for delayed_block_number, new_fee_multiplier in [
```

**Review by roy-dydx — APPROVED** [review]

File: protocol/scripts/governance/bridge_vesters_set_rewards.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+            "authority": AUTHORITY,
+            "msg": {
+                "@type": "/dydxprotocol.rewards.MsgUpdateParams",
+                "authority": DELAY_MSG_MODULE_ADDRESS,
+                "params": {
+                    "treasuryAccount": "rewards_treasury",
+                    "denom": NATIVE_TOKEN_DENOM,
+                    "denomExponent": -18,
+                    "marketId": 1000001,
+                    "fee_multiplier_ppm": fee_multiplier
```
Comment (ttl33) [inline protocol/scripts/governance/bridge_vesters_set_rewards.py:152]:
```suggestion
                    "fee_multiplier_ppm": new_fee_multiplier
```

File: protocol/scripts/governance/bridge_vesters_set_rewards.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+                }
+            },
+            "delay_blocks": delay
```
Comment (ttl33) [inline protocol/scripts/governance/bridge_vesters_set_rewards.py:155]:
```suggestion
            "delay_blocks": delayed_block_number
```

**Review by ttl33 — APPROVED** [review]
all nits, overall lgtm

**Pull request merged**